main.py

```diff
@@ -1,11 +1,36 @@
 import init_django_orm  # noqa: F401
-
+import json
 from db.models import Race, Skill, Player, Guild
 
 
 def main() -> None:
-    pass
+    with open("players.json") as file:
+        players_info = json.load(file)
+
+    for nick, info in players_info.items():
+        race_info = info["race"]
+        guild_info = info["guild"]
+
+        get_rase, _ = Race.objects.get_or_create(
+            name=race_info["name"],
```

Comment:
`get_rase` looks like func name , maybe call this viriable `race_data`  or smth like that . And yeah raSe ? Did you mean raCe ?)